example/src/main/java/io/netty/example/http/websocketx/server/WebSocketIndexPageHandler.java

```diff
@@ -56,7 +56,7 @@ protected void channelRead0(ChannelHandlerContext ctx, FullHttpRequest req) thro
         }
 
         // Allow only GET methods.
-        if (!GET.equals(req.method())) {
+        if (!GET.toString().equals(req.method().toString())) {
```

Comment:
@huihaoweishui make this `HttpMethod.GET.equals(req.method())` to eliminate extra String creations

Reply:
@amizurov no... we don't call `toString` in my suggestion.

Reply:
> > @huihaoweishui make this `HttpMethod.GET.equals(req.method())` to eliminate extra String creations
> 
> But we already have this code in origin or I something missed :)

missed indeed